onadata/libs/models/abstract_base.py

```diff
@@ -0,0 +1,13 @@
+"""AbstractBase model"""
+
+from django.db import models
+
+
+class AbstractBase(models.Model):
+    """Common information for models"""
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
```

Comment:
I am a bit confused about this class and especially in terms of naming as compared to Python [Abstract Base Clasess](https://docs.python.org/3/library/abc.html). Could this have been named differently?

Also, it does create a new way for the fields, which, in this codebase, we have used `date_created` and `date_modified`. I could think of two other user-related fields that could be considered part of this.

Reply:
@ukanga Sure, we can rename the fields. Also I have actually found we already have an exsisting base model, we can use that https://github.com/onaio/onadata/blob/main/onadata/libs/models/base_model.py